go/vt/vtadmin/cluster/cluster.go

```diff
@@ -353,6 +354,31 @@ func (c *Cluster) parseTablet(rows *sql.Rows) (*vtadminpb.Tablet, error) {
 	return tablet, nil
 }
 
+// ApplySchema applies a schema to the given keyspace in this cluster.
+func (c *Cluster) ApplySchema(ctx context.Context, req *vtctldatapb.ApplySchemaRequest) (*vtctldatapb.ApplySchemaResponse, error) {
+	span, ctx := trace.NewSpan(ctx, "Cluster.ApplySchema")
+	defer span.Finish()
+
+	AnnotateSpan(c, span)
+	span.Annotate("keyspace", req.Keyspace)
+	span.Annotate("sql", strings.Join(req.Sql, "; "))
+	span.Annotate("ddl_strategy", req.DdlStrategy)
+	span.Annotate("uuid_list", strings.Join(req.UuidList, ", "))
+	span.Annotate("migration_context", req.MigrationContext)
+
+	if d, ok, err := protoutil.DurationFromProto(req.WaitReplicasTimeout); ok && err != nil {
```

Comment:
The default value is nil, so I'd check for that and use the zeroval when it is.

Reply:
Never mind. Resolving this as I can see that the nil case is handled properly within the function:
```
func DurationFromProto(dpb *vttime.Duration) (time.Duration, bool, error) {
	if dpb == nil {
		return 0, false, nil
	}
```
```